pkg/servicemanager/manager.go

```diff
@@ -122,6 +124,7 @@ func (m *ServiceManager) asyncRun(ctx context.Context, service Service) (<-chan
 			go func() {
 				<-ready
 				klog.InfoS("SERVICE READY", "service", service.Name(), "since-start", time.Since(svcStart))
+				startLog.LogService(service.Name(), service.Dependencies(), svcStart, time.Now())
```

Comment:
We can think about moving the log above to this function.
We can also think about renaming the function to something like `LogReadyService` or `LogServiceReadiness`.

Reply:
I think that's a good idea to have it in one place, changed

Reply:
Also, renamed the function to `LogServiceReady` to stay consistent with other functions in the package